src/store/task/actions/taskActions.js

```diff
@@ -1,9 +1,14 @@
-import { ADD_TASK, DELETE_TASK } from "src/store/task/types/taskTypes";
-import { formatDate } from "src/utils/formatDate";
+import {
+  ADD_TASK,
+  DELETE_TASK,
+  DONE_TASK,
+} from "src/store/task/types/taskTypes";
 
 export const addTask = (task) => {
   task.taskId = Date.now().toString();
-  task.createdAt = formatDate(new Date());
```

Comment:
we don't need to format date here anymore?

Reply:
@ray-wtag vai, no we don't need to format here anymore. We are now storing createdAt as date directly, as it gets very difficult to compare two dates otherwise.

We are formatting the in UI however to match that according to the design.